lib/Dialect/CGGI/Transforms/Passes.td

```diff
@@ -19,4 +19,20 @@ def SetDefaultParameters : Pass<"cggi-set-default-parameters"> {
   let dependentDialects = ["mlir::heir::cggi::CGGIDialect"];
 }
 
+def BooleanLineVectorizer : Pass<"cggi-boolean-line-vectorize"> {
+  let summary = "Group different logic gates with the packed API";
+  let description = [{
+    This pass groups indepentend logic gates into a single call of the packed
+    operations.
+
+    Pass is intended for the `FPT` tfhe-rs API.
```

Comment:
nit: might benefit from a short example, or some description that this is an improvement over straight line vectorizer.